src/plugins/usage_collection/server/config.ts

```diff
@@ -19,16 +19,10 @@
 
 import { schema, TypeOf } from '@kbn/config-schema';
 import { PluginConfigDescriptor } from 'kibana/server';
-import { DEFAULT_MAXIMUM_WAIT_TIME_FOR_ALL_COLLECTORS_IN_S } from '../common/constants';
 
 export const configSchema = schema.object({
-  uiMetric: schema.object({
-    enabled: schema.boolean({ defaultValue: true }),
-    debug: schema.boolean({ defaultValue: schema.contextRef('dev') }),
-  }),
-  maximumWaitTimeForAllCollectorsInS: schema.number({
-    defaultValue: DEFAULT_MAXIMUM_WAIT_TIME_FOR_ALL_COLLECTORS_IN_S,
-  }),
+  enabled: schema.boolean({ defaultValue: true }),
```

Comment:
There is the risk that disabling `uiMetric`, we'll disable this entire plugin (the core service would still be there though)?